Engine/Module/ModuleRender.cpp

```diff
@@ -158,6 +158,10 @@ void ModuleRender::Render() const
 	if (App->cameras->main_camera != nullptr) 
 	{
 		App->cameras->main_camera->RecordFrame(App->window->GetWidth(), App->window->GetHeight());
+		App->cameras->directional_light_camera->RecordFrame(App->window->GetWidth(), App->window->GetHeight());
+		App->cameras->directional_light_mid->RecordFrame(App->window->GetWidth(), App->window->GetHeight());
+		App->cameras->directional_light_far->RecordFrame(App->window->GetWidth(), App->window->GetHeight());
```

Comment:
In game the three different directional light frustums have the same resoultion, while in `PanelScene` and `PanelGame` don't. Why is that?